app/models/user_history.rb

```diff
@@ -85,7 +85,8 @@ def self.actions
       entity_export: 66,
       change_password: 67,
       topic_timestamps_changed: 68,
-      approve_user: 69
+      approve_user: 69,
+      webhook: 70
```

Comment:
```suggestion
      web_hook: 70
```

Reply:
Not sure why but we named the model `WebHook`. I think we should keep the consistency here.